src/Uno.UI/UI/Xaml/UIElement.skia.cs

```diff
@@ -52,16 +52,14 @@ public UIElement()
 
 		public bool UseLayoutRounding
 		{
-			get => (bool)this.GetValue(UseLayoutRoundingProperty);
```

Comment:
I'm surprised that reading the property is expensive and curious to know what part of the DP system is showing up in traces.

Reply:
NOTE: Given that the property is `MetaDataPropertyInfoFlags::IsPropMethodCall` in WinUI, this change is fine and reasonable until Uno implements `IsPropMethodCall` (tracked by https://github.com/unoplatform/uno/issues/17300). After `IsPropMethodCall` is implemented, you can try using it and see if the perf is still a concern.

It would still be good to understand exactly where the perf hit is, and what scenario is being profiled.

Reply:
> some numbers from before/after traces would be nice IMO

That we agree on :)

Reply:
This DP is read on Measure/Arrange/Clip for Borders/Grids/TextBlocks/ScrollViewer/...

Numbers are measured on a synthetic bench (Dopes Change) for 11s ~, real traffic would be less but it still piles up.

Before:

![image](https://github.com/user-attachments/assets/a4558319-00ac-4d30-a599-8968514b691e)

After:

![image](https://github.com/user-attachments/assets/cdd2e56c-afd5-4f2a-a394-18d0dae94609)